app/schemas/com.jerboa.db.AppDB/18.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 18,
-    "identityHash": "214278f3dfc312f44830b58678afc3eb",
+    "identityHash": "9a881a3925f19533ba5f6d339f9d0461",
```

Comment:
It seems like this whole file shouldn't exist yet. @MV-GH do you know what's up here?

Reply:
Yeah that was a mistake, it was added already by accident. It was a leftover from testing #871